extensions/vscode/src/commands/make-brick.ts

```diff
@@ -0,0 +1,149 @@
+import { lstatSync } from "fs";
+import * as vscode from "vscode";
+import {
+  Uri,
+  window
+} from "vscode";
+import { masonList, masonInfo, masonMake } from "../mason";
+
+export const makeBrick = async (uri: Uri) => {
+  if (!lstatSync(uri.fsPath).isDirectory()) {
+    window.showErrorMessage("Please select a valid directory");
+    return;
+  }
+
+  const cwd = vscode.workspace.workspaceFolders?.[0].uri.fsPath;
+  if (!cwd) {
+    return;
+  }
+
+  const targetDirectory = uri.fsPath;
+  if (!targetDirectory) {
+      return;
+  }
+
+  const bricks = await getListOfBricks({ cwd });
+  if (!bricks) {
+    window.showErrorMessage("No bricks found in the workspace");
+    return;
+  }
+
+  const brickName = await promptForBrickName({ bricks });
+  if (!brickName) {
+      return;
+  }
+
+  const rawBrick = await masonInfo({ cwd, brickName });
```

Comment:
I think we should be able to parse the cached bricks rather than adding another command to the mason_cli. That should drastically simplify this PR and I’m happy to make that change if it sounds good to you.

Reply:
Hey Felix! 👋
Sure, that sounds great!
Probably I should have been spending more time investigating the current implementation.😅

Reply:
No worries I can take it from here and get everything merged later today. Thanks again for this amazing contribution!